### PR TITLE
use rate limiter counter for more detailed traffic stats

### PR DESCRIPTION
**Pull request by ShanniLi**
r @Raynos @kriskowal @anson627

### CODE REVIEW

File: node/hyperbahn/service_proxy.js [hunk 1]
```diff
@@ -124,6 +124,9 @@ ServiceDispatchHandler.prototype.rateLimit =
 function rateLimit(req, buildRes) {
     var self = this;
 
+    // stats all traffic using rate limiter's counter
+    self.rateLimiter.incrementServiceCounter(req.headers.cn + '~~' + req.serviceName);
```
Comment (Raynos) [inline node/hyperbahn/service_proxy.js]: I think it's bad to make this a service counter. 

I think its better to have a new bucket in the rate limiter thats not "ServiceCounter" but is instead "EdgeCounter"

Comment (Raynos) [inline node/hyperbahn/service_proxy.js]: It will also be good to have different stats for service-rps and edge-rps

Comment (Raynos) [inline node/hyperbahn/service_proxy.js]: We also shouldnt rate limit edges. 

This is a lot of coupling and nuance here.

**Comment by Raynos** [comment]
This needs more work and less coupling.

File: node/hyperbahn/service_proxy.js (hunk 1 above)
Comment (ShanniLi) [inline node/hyperbahn/service_proxy.js]: the counter is not rate limited. I will make changes to decouple.

**Comment by Raynos** [comment]
lgtm.

**Pull request merged**